src/MSGTBL.CPP

```diff
@@ -24,7 +24,7 @@
 #include <messages.h>	// MSGTBL,
 
 //===========================================================================
-MSGTBL far msgTbl[] =		//(far rob 1-92 for BC)
```

Comment:
Why did "far" disappear entirely, instead of becoming "FAR?" 
Also, if it's supposed to disappear, then the comment afterward should probably go too.